airbyte-integrations/connectors/source-confluence/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_confluence ./source_confluence
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.3
+LABEL io.airbyte.version=0.1.0
```

Comment:
Can you update the version of the connector appropriately here? It looks like you're downgrading here.